datafusion/proto/tests/cases/roundtrip_logical_plan.rs

```diff
@@ -591,10 +591,6 @@ async fn roundtrip_expr_api() -> Result<()> {
             lit(2),
         ),
         array_positions(make_array(vec![lit(4), lit(3), lit(3), lit(1)]), lit(3)),
-        array_except(
-            make_array(vec![lit(1), lit(2), lit(3)]),
-            make_array(vec![lit(1), lit(2)]),
-        ),
```

Comment:
I think you accidentally remove it. Also, add remove's too

Reply:
Thanks for the review, addressed.